pkg/nodeconfig/nodeconfig.go

```diff
@@ -64,6 +65,8 @@ type nodeConfig struct {
 	additionalAnnotations map[string]string
 	// additionalLabels are extra labels that should be applied to configured nodes
 	additionalLabels map[string]string
+	// platformType holds the name of the platform where cluster is deployed
+	platformType configv1.PlatformType
```

Comment:
I'm not seeing this used anywhere by nodeconfig in this PR. In that case it should be removed from the struct and only passed into windows.New().

Reply:
Yep, while that is the case, as a rule of thumb, I'd prefer to only introduce struct changes in the same PR they are being utilized. For reasons such as plans changing when working on the other PR, and making it easier to revert changes, if that becomes necessary.

With the nodeconfig struct change this PR will be fine to merge after #808 merges, or it can be merged before, with this change removed.

Reply:
ack, it's fixed now